migration/src/test/java/org/axonframework/migration/AxonJavaxTest.java

```diff
@@ -0,0 +1,42 @@
+package org.axonframework.migration;
```

Comment:
I'm missing the copyright notice in this file. You can find the notice template [here](https://github.com/AxonFramework/AxonFramework/blob/master/copyright-template.xml)

Reply:
Added.